test/Kestrel.Core.Tests/FrameTests.cs

```diff
@@ -335,7 +335,9 @@ public void InitializeStreamsResetsStreams()
             string requestLine,
             string expectedMethod,
             string expectedRawTarget,
+#pragma warning disable xUnit1026 // Theory methods should use all of their parameters
```

Comment:
Comment that it's sharing theory data with with HttpParserTests.ParsesRequestLine.